src/types/package.rs

```diff
@@ -6,6 +6,7 @@ use std::fmt;
 use std::str::FromStr;
 
 use chrono::{DateTime, Utc};
+use ordered_float::NotNan;
```

Comment:
It seems heavy handed to add a dependency just for this... And it doesn't seem like we're getting much benefit. The JSON schema still just describes this value as a float. And the deserialization guarantee doesn't matter much since we're not doing any arithmetic with this value as far as I know.